### PR TITLE
Fix seek seconds.

### PR DESCRIPTION
**Pull request by yschimke**
#### WHAT

image

#### WHY

Show seconds

#### Checklist :clipboard:
- [x] Add explicit visibility modifier and explicit return types for public declarations
- [x] Run spotless check
- [x] Run tests
- [x] Update metalava's signature text files

### CODE REVIEW

File: media-ui/src/main/java/com/google/android/horologist/media/ui/components/controls/SeekButtonIncrement.kt [hunk 1]
```diff
@@ -32,4 +31,15 @@ public sealed class SeekButtonIncrement(
     public object Thirty : SeekButtonIncrement(30)
 
     public data class Other(override val seconds: Int) : SeekButtonIncrement(seconds)
+
+    public companion object {
+        public fun SeekButtonIncrement(seconds: Int): SeekButtonIncrement {
```
Comment (luizgrp) [inline media-ui/src/main/java/com/google/android/horologist/media/ui/components/controls/SeekButtonIncrement.kt:36]: nit: could make seconds param nullable and return Unknown in that scenario?

**Review by luizgrp — APPROVED** [review]

File: media-ui/src/main/java/com/google/android/horologist/media/ui/components/controls/SeekButtonIncrement.kt (hunk 1 above)
Comment (yschimke) [reply media-ui/src/main/java/com/google/android/horologist/media/ui/components/controls/SeekButtonIncrement.kt:36]: https://github.com/google/horologist/issues/308

Comment (luizgrp) [reply media-ui/src/main/java/com/google/android/horologist/media/ui/components/controls/SeekButtonIncrement.kt:36]: hmm I meant this factory function and not the interface itself

btw this function name should start with lower case, it gets confusing after naming so many composable functions that way 😅

Comment (luizgrp) [reply media-ui/src/main/java/com/google/android/horologist/media/ui/components/controls/SeekButtonIncrement.kt:36]: weird that code analysis is not picking it up

Comment (yschimke) [reply media-ui/src/main/java/com/google/android/horologist/media/ui/components/controls/SeekButtonIncrement.kt:36]: Yep, I understood about the factory taking a nullable, but the problem is the input isn't nullable so doesn't help yet. I've done it now, but will still need to follow up and remove the default of 0.

Comment (yschimke) [reply media-ui/src/main/java/com/google/android/horologist/media/ui/components/controls/SeekButtonIncrement.kt:36]: BTW this does follow the conventions

_Exception: factory functions used to create instances of classes can have the same name as the abstract return type:_

```
interface Foo { /*...*/ }

class FooImpl : Foo { /*...*/ }

fun Foo(): Foo { return FooImpl() }
```

Comment (luizgrp) [reply media-ui/src/main/java/com/google/android/horologist/media/ui/components/controls/SeekButtonIncrement.kt]: thanks!! TIL

**Pull request merged**